app/controllers/MissionController.scala

```diff
@@ -108,18 +106,14 @@ class MissionController @Inject() (implicit val env: Environment[User, SessionAu
     )
   }
 
-  /**
+  /** If the dist a user has audited in a region implies that they should have completed more missions, add them.
     *
     * @param userId
     * @param regionId
     */
-  def updatedUnmarkedCompletedMissionsAsCompleted(userId: UUID, regionId: Int): Unit = {
+  def updateUnmarkedCompletedMissionsAsCompleted(userId: UUID, regionId: Int): Unit = {
     val missions = MissionTable.selectIncompleteMissionsByAUser(userId, regionId)
-    val streets = StreetEdgeTable.selectStreetsAuditedByAUser(userId, regionId)
-    val CRSEpsg4326 = CRS.decode("epsg:4326")
-    val CRSEpsg26918 = CRS.decode("epsg:26918")
-    val transform = CRS.findMathTransform(CRSEpsg4326, CRSEpsg26918)
-    val completedDistance_m = streets.map(s => JTS.transform(s.geom, transform).getLength).sum
+    val completedDistance_m = StreetEdgeTable.getDistanceAudited(userId, regionId)
 
```

Comment:
Switches from JTS.transform to calling a function in StreetEdgeTable that calculate the distance in the query.